guides/bug_report_templates/action_view_gem.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  gem "rails", "~> 7.1.0"
+end
+
+require "minitest/autorun"
+require "action_view"
+
+class BugTest < ActionView::TestCase
+  module Helpers
+    def upcase(value)
+      value.upcase
+    end
+  end
+
+  def test_stuff
+    view.extend Helpers
```

Comment:
This isn't how you add helpers to test cases, you use the `helper` class method.

Reply:
Thank you for calling that out! Is there somewhere in the documentation I could learn more about that? I can't find the method in [the API docs](https://api.rubyonrails.org/v7.1.1/classes/ActionView/TestCase.html)

Reply:
It is this method https://api.rubyonrails.org/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper

Reply:
@rafaelfranca I've pushed up a change to use that method.